.ci/deps.sh

```diff
@@ -56,6 +56,10 @@ if [[ "x${PLUGIN}" == "xsource/mysql" ]] || \
   docker pull mariadb:10
 fi
 
+if [[ "x${PLUGIN}" == "xmodel/tensorflow" ]];
+  python -m pip install scikit-learn
```

Comment:
Why was this done?

Reply:
For the tensorflow tests one of them was failing due to this error, I looked into the logs and found that sklearn was not getting installed so, I added the sklearn module to be installed for the tests.
```console
======================================================================
ERROR: tests.test_tf_integration (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_tf_integration
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/runner/work/dffml/dffml/model/tensorflow/tests/test_tf_integration.py", line 11, in <module>
    from sklearn.datasets import make_classification, make_regression
ModuleNotFoundError: No module named 'sklearn'
```